test/opinionatedtest.jl

```diff
@@ -0,0 +1,58 @@
+@testset "opinionatedtest" begin
+
+    # unit testing 
+    #simple test case
+    @test true

```

Comment:
this `@test true` is totally unneeded. However I am having trouble to release what is the purpose of this testing file. It looks more or less like a repetition from the `./test/attributes.jl` (?)